nfc/libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -264,7 +408,6 @@ phNciNfc_ReInitialise(void* pNciHandle,
         pNciContext->ResetInfo.ResetTypeReq = phNciNfc_ResetType_KeepConfig;
         /* Init payload to display the build number in the init response */
         pNciContext->tInitInfo.bExtension = 0x00;
-        pNciContext->tInitInfo.bSkipRegisterAllNtfs = 0x01;
```

Comment:
The idea of Remove "bSkipRegisterAllNtfs from phNciNfc_InitInfo" is to remove the flag "pNciContext->tInitInfo.bSkipRegisterAllNtfs".
This leads to call phNciNfc_RegAllNtfs when needed; before the NCI initialization sequence start, in order to be able catch any early notification.

I agree, the comment is lite or not existing at all.
If you agree with the idea, i will add a comment to clarify the change.